admin_guide/overcommit.adoc

```diff
@@ -74,7 +74,25 @@ filter-pre-mux-client.conf:
   buffer_path `/var/lib/fluentd/buffer-mux-client`
 ----
 
-The Fluentd `buffer_queue_limit` is 32.
+The Fluentd `buffer_queue_limit` is the value of the variable `BUFFER_QUEUE_LIMIT`. This value is `32` by default.
+
+The environment variable `BUFFER_QUEUE_LIMIT` is calculated as `(FILE_BUFFER_LIMIT / (number_of_outputs * BUFFER_SIZE_LIMIT))`.
+
+In this formula, `number_of_outputs` is `1` if all the logs are sent to a single resource and it is incremented by `1` for each additional resource. For example, the value of `number_of_outputs` will be:
+
+ * `1` - If all logs are sent to a single ElasticSearch pod.
+ * `2` - If application logs are sent to an ElasticSearch pod and ops logs are sent to
+another ElasticSearch pod.
+ * `4` - If application logs are sent to an ElasticSearch pod and ops logs are sent to
+another ElasticSearch pod. And both are forwarded to other Fluentd instances.
+
+Therefore, if `BUFFER_QUEUE_LIMIT` has the default set of values:
```

Comment:
Are these the default values? I'd say "By default, these parameters are assigned the following values:"

I'd move lines 89-95 to line 80, right after you present the formula.